peers/app_request_network.go

```diff
@@ -55,7 +55,7 @@ func NewNetwork(
 		),
 	)
 
-	metrics, err := NewAppRequestNetworkMetrics(cfg, registerer)
+	metrics, err := newAppRequestNetworkMetrics(registerer)
```

Comment:
I can't leave a comment at the exact line, but let's also measure the latency of the call to `infoAPI.GetNetworkID` in the constructor.

Reply:
Looks like this comment is not resolved, but on second thought I think it's fine to skip measuring this call since it's only called once at startup.